circle.yml

```diff
@@ -70,7 +70,7 @@ executors:
   macos-xcode-min:
     resource_class: macos.x86.medium.gen2
     macos:
-      xcode: 13.4.1
+      xcode: 14.3.0
```

Comment:
I don't think it make sense to have `14.3.0` as minimal and `14.3.1` as default. Let's stick to `14.3.1` here. But we can disable `macos-xcode-min` build for now (just add TODO to re-enable it in case we upgrade Xcode again.

Reply:
done